src/components/o-modal/index.js

```diff
@@ -51,9 +51,7 @@ class AXAModal extends BaseComponentGlobal {
     const axaButtonsRaw = this.querySelectorAll('axa-button[data-modal-close]');
     const axaButtons = Array.from(axaButtonsRaw);
     if (axaButtons && axaButtons.length) {
-      axaButtons.forEach((button) => {
-        button.addEventListener('click', this.closeModalFunction);
-      });
+      axaButtons.forEach(button => button.addEventListener('click', this.closeModalFunction));
```

Comment:
Why no event delegation?

Reply:
Good point. IMHO, if I do not expect a ton of buttons in there, readability goes over performance. Usually, I guess a modal will have two buttons mostly, maybe in some cases ten, who knows. Therefore I would go for readability. IMHO this is more readable than the complexity layer that event delegation would add.